specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,132 @@
+                "operationId": "AllowedConnections_ListByHomeRegion",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/AscLocation"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/AllowedConnectionsList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/locations/{ascLocation}/allowedConnections/{connectionsType}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get allowed connections": {
+                        "$ref": "./examples/AllowedConnections/GetAllowedConnections_example.json"
+                    }
+                },
+                "tags": [
+                    "AllowedConnections"
+                ],
+                "description": "Gets a specific allowed connections component.",
+                "operationId": "AllowedConnections_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "$ref": "#/parameters/AscLocation"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionsType"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/AllowedConnectionsResource"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }

```

Comment:
@yanivn3

Reply:
Chen, list by RG is implicitly defined when we have list by Susbcription, ins't ? We do not have in other API's list by RG (although supported)

Reply:
@yanivn3 if it's not defined in the swagger it will not be documented and will not be available to any generated client. if the path is supported then it should be available in the swagger.